Kitodo-API/src/main/java/org/kitodo/api/validation/longtermpreservation/LtpValidationCondition.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.api.validation.longtermpreservation;
+
+import java.util.List;
+
+/**
+ * An immutable, no-setters beans implementing the validation condition interface.
+ */
+public class LtpValidationCondition implements LtpValidationConditionInterface {
```

Comment:
@solth I think the use of abbreviations in individual cases, such as `Ltp` here, is perfectly fine, but it deviates from the current concept of spelling out all names in full. I'm **in favor of** using _some_ abbreviations, and perhaps being a bit more relaxed in the future, because the names have sometimes become extremely cumbersome. But I'll leave the final decision to you.